src/r5py/r5/transport_network.py

```diff
@@ -139,7 +114,7 @@ def __exit__(self, exception_type, exception_value, traceback):
     def _cache_directory(self):
         cache_dir = (
             pathlib.Path(Config().TEMP_DIR)
-            / f"{self.__class__.__name__:s}_{hash(self):x}"
+            / f"{self.__class__.__name__:s}_{id(self):x}"
```

Comment:
changed to the definitely unique `id(self)`, instead of implementing an own `__hash__()`. I wanted to make sure we don’t run into filename collisions, and wanted to be on the safe side